packages/frontend/src/components/TableConfigPanel/ConditionalFormattingList.tsx

```diff
@@ -0,0 +1,114 @@
+
+const ConditionalFormattingList = ({}) => {
+    const {
+        explore,
+        tableConfig: { conditionalFormattings, onSetConditionalFormattings },
+    } = useVisualizationContext();
+
+    const activeFields = useExplorerContext((c) => c.state.activeFields);
+
+    const visibleActiveNumericFields = useMemo(() => {
+        if (!explore) return [];
+
+        return getVisibleFields(explore)
+            .filter((field) => activeFields.has(fieldId(field)))
+            .filter((field) => field.type === 'number');
+    }, [explore, activeFields]);
+
+    const activeConfigs = useMemo(() => {
+        return conditionalFormattings.filter((config) =>
+            config.target
+                ? visibleActiveNumericFields.find(
+                      (field) => fieldId(field) === config.target?.fieldId,
+                  )
+                : true,
+        );
+    }, [conditionalFormattings, visibleActiveNumericFields]);
+
+    const usedFieldIds = useMemo(() => {
+        return activeConfigs
+            .map((c) => c.target?.fieldId)
+            .filter((f): f is string => !!f);
+    }, [activeConfigs]);
+
+    const handleAdd = useCallback(() => {
+        onSetConditionalFormattings(
+            produce(activeConfigs, (draft) => {
+                draft.push(createConditionalFormattingConfig());
+            }),
+        );
+    }, [onSetConditionalFormattings, activeConfigs]);
+
+    const handleRemove = useCallback(
+        (index) =>
+            onSetConditionalFormattings(
+                produce(activeConfigs, (draft) => {
+                    draft.splice(index, 1);
+                }),
+            ),
+        [onSetConditionalFormattings, activeConfigs],
+    );
+
+    const handleOnChange = useCallback(
+        (index, newConfig) =>
+            onSetConditionalFormattings(
+                produce(activeConfigs, (draft) => {
+                    draft[index] = newConfig;
+                }),
+            ),
+        [onSetConditionalFormattings, activeConfigs],
+    );
+
+    return (
+        <ConditionalFormattingListWrapper>
+            {activeConfigs.map((conditionalFormatting, index) => (
+                <ConditionalFormatting
+                    key={index}
+                    index={index}
+                    fields={visibleActiveNumericFields}
+                    usedFieldIds={usedFieldIds}
+                    value={conditionalFormatting}
+                    onChange={(newConfig) => handleOnChange(index, newConfig)}
+                    onRemove={() => handleRemove(index)}
+                />
+            ))}
+
+            <FormGroup>
+                <Tooltip2
+                    position="bottom-left"
+                    disabled={
+                        visibleActiveNumericFields.length !==
+                        usedFieldIds.length
+                    }
+                    content="All fields are already being used in rules."
```

Comment:
Blueprint has some issues with tooltips on disabled elements. 


https://user-images.githubusercontent.com/9117144/212322848-ce1118ad-ca12-4369-9101-da6b5ebde099.mp4

Reply:
In `packages/frontend/src/components/RefreshButton.tsx` we only used the disabled class and prevented the onClick action.

```
                // disabled button captures hover events
                onClick={isButtonDisabled ? undefined : onClick}
                className={isButtonDisabled ? Classes.DISABLED : undefined}
            ```

Reply:
that worked perfectly, thanks!